materialize-mysql/.snapshots/TestConfigURI-IncorrectSSL

```diff
@@ -1,2 +1,2 @@
-will:secret1234@tcp(example.com:3306)/somedb?tls=
+will:secret1234@tcp(example.com:3306)/somedb
```

Comment:
Is the change here significant? I'm assuming that `tls=` is equivalent to not having that parameter at all, but just wanted to double check.

Reply:
This output can only be reached if an invalid SSL mode is selected, which will cause a validation error, so we'd never actually try to connect with this DSN. On a purely academic level, I'd suspect that omitting the parameter entirely is more along the lines of what MySQL wants, since that is the output from the mysql library `FormatDSN()` method when the TLS config is set to an empty string.